README.md

```diff
@@ -19,6 +19,10 @@
 
 OpenShift Do (Odo) is a CLI tool for developers who are writing, building, and deploying applications on OpenShift. With Odo, developers get an opinionated CLI tool that supports fast, iterative development which abstracts away Kubernetes and OpenShift concepts, thus allowing them to focus on what's most important to them: code.
 
+### Demo
+
+![demo](/docs/img/example.gif)
```

Comment:
why we are not using asciinema link directly ?

Reply:
Because asciinema won't play automatically within Markdown. You'd have to click the link and press play twice, disrupting the user experience for the README...

Reply:
Personally, I don't mind that much but if we think it really improves thing, let's go ahead with including the gif.